test-config.yaml

```diff
@@ -0,0 +1,21 @@
+accounts:
+  - mnemonic: "satisfy adjust timber high purchase tuition stool faith fine install that you unaware feed domain license impose boss human eager hat rent enjoy dawn"
+    name: validator
+    coins: ["20000snr", "100000000stake"]
+    # address: snr1phaxpevm5wecex2jyaqty2a4v02qj7qmqe4lvm
+  - mnemonic: "notice oak worry limit wrap speak medal online prefer cluster roof addict wrist behave treat actual wasp year salad speed social layer crew genius"
+    name: test1
+    coins: ["20000snr", "200000000stake"]
+    # address: snr1cyyzpxplxdzkeea7kwsydadg87357qnaglp5wk
```

Comment:
I think Brayden and Josh talked during our demo about swapping out account types. Did that happen?

Reply:
is putting mnemonic in config safe? could this be an env variable in github? @lakshya-sky @ntindle

Reply:
@ntindle This accounts are needed to run the cosmos chain with the same address during restarts, I don't think we can do MPC related stuff in here.
@prnk28 This are completely safe, since we don't use them in production, also these are taken from Terra dev configs.

Reply:
@mcjcloud

Reply:
I think this is fine for now, once we get MPC figured out better we can come back and refactor this.